### PR TITLE
Fixing calculation of shift of slur position

### PR DESCRIPTION
**Pull request by valeriyvan**
Not sure my fix is right. Hope somewhere close.

### CODE REVIEW

File: src/slur.cpp [hunk 1]
```diff
@@ -364,18 +363,18 @@ void Slur::AdjustSlurPosition(Doc *doc, FloatingCurvePositioner *curve,
...
 
-        shift = 0;
         // Keep the maximum shift on the left and right
+        int shift;
         if (curveDir == curvature_CURVEDIR_above) {
             shift = intersection;
         }
         else {
-            shift = intersection;
+            shift = -intersection;
```
Comment (lpugin) [inline src/slur.cpp]: Why are you changing this?

Comment (valeriyvan) [reply src/slur.cpp]: I don't understand logic of calculation. 
But doing same in both branches of `if` suggests here is a bug. 
I thought it was meant something like that.
No? What should it be then?

Comment (lpugin) [reply src/slur.cpp]: OK. I have to look at this because we need to make sure we fix it in the right direction.

Comment (rettinghaus) [reply src/slur.cpp]: @lpugin what's with this PR?

Comment (valeriyvan) [reply src/slur.cpp]: Nothing from my side. Current calculation is definitely wrong and has a bug. But I am unable to fix it.

Comment (lpugin) [reply src/slur.cpp]: I can see that doing then same in a if/else makes no sense, but could provide us with an MEI snippet that illustrates the bug?

Comment (valeriyvan) [reply src/slur.cpp]: No, I can't provide MEI. Having such MEI I would probably have hints how to fix.

**Comment by valeriyvan** [comment]
Fixed with PR #1506

**Pull request closed without merging**